ci/functional/test_main.sh

```diff
@@ -2,13 +2,13 @@
 
 set -eux
 
-test_tag=$(git show -s --format=%B | \
-           sed -ne "/^Test-tag$PRAGMA_SUFFIX:/s/^.*: *//p")
-if [ -z "$test_tag" ]; then
-    # shellcheck disable=SC2153
-    test_tag=$TEST_TAG
+if [ -z "$TEST_TAG" ]; then
```

Comment:
(lint) Possible misspelling: TEST_TAG may not be assigned, but test_tag is. [SC2153]